commands/roll.js

```diff
@@ -52,7 +61,12 @@ module.exports = {
             rollEmbed.setColor('#990000')
         else
             rollEmbed.setColor('#999999')
-        rollEmbed.addField(bot.rollToDetails(rollData), bot.rollToSpoiler(rollData));
+        // set up options
+        const options = [];
+        if (!spoilers) options.push("nospoilers");
+        rollEmbed.addField(bot.rollToDetails(rollData), bot.rollToSpoiler(rollData, WhisperType.NO, options));
+        if (plaintext) options.push("plaintext");
+        rollEmbed.addField(bot.rollToDetails(rollData, options), bot.rollToSpoiler(rollData, WhisperType.NO, options));
```

Comment:
oups, I missed that this was adding a call to `addField` twice.. probably due to the merge conflict being manually resolved.
It caused all rolls to appear twice. Noticed it while testing the plaintext option
![image](https://user-images.githubusercontent.com/27990/198858995-e704c587-4005-4f71-a096-94089044a493.png)
Fixed now, and I've deployed the changes.
Thanks!